bin/prepare-kubectl

```diff
@@ -3,5 +3,5 @@
 mkdir -p ~/.kube
 
 if [ -n "$KUBECONFIG_DATA" ]; then
-  echo "${KUBECONFIG_DATA}" | base64 --decode > "${KUBECONFIG:-~/.kube/config}"
+  echo "${KUBECONFIG_DATA}" | base64 --decode > ${KUBECONFIG:-~/.kube/config}
```

Comment:
How about `echo "${KUBECONFIG:-${HOME}/.kube/config}"` instead (so we can keep the double-quotes).

Reply:
Is $HOME always going to be available?